xbmc/platform/posix/main.cpp

```diff
@@ -41,6 +44,42 @@
 #include "input/linux/LIRC.h"
 #endif
 #include "platform/XbmcContext.h"
+  
+#if defined(TARGET_POSIX)
```

Comment:
Yes, I also thought that, but decided to follow the existing code here which already does this.
Should I remove the existing #ifdef TARGET_POSIX here, too?